common/holders/getAccountStateOptions.go

```diff
@@ -0,0 +1,25 @@
+package holders
+
+type getAccountStateOptions struct {
+	blockRootHash []byte
+}
+
+// NewGetAccountStateOptions creates a getAccountStateOptions
+func NewGetAccountStateOptions(blockRootHash []byte) *getAccountStateOptions {
```

Comment:
a more appropriate name would be, rootHashHolder as we already have the function ) GetBlockRootHash()
And speaking of naming, I thing ) GetBlockRootHash() might become ) GetRootHash() (as we can reuse this component even on data tries, if we somehow need it)

Reply:
Originally, I've added the holder to wrap one or more parameters necessary to `get account at given block`. But I see your point. Fixed.